app/Http/Controllers/Admin/AdminHotelController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Hash;
+use DB;
+use Session;
+use HotelBooking\Hotel;
+use HotelBooking\AdminHotel;
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Requests\Admin\AdminHotelCreateFormRequest;
+use HotelBooking\Http\Requests\Admin\AdminHotelUpdateFormRequest;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Controllers\Admin\AdminBaseController;
+
+class AdminHotelController extends AdminBaseController
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        
+    }
+
+    /**
+     * Show the form for creating a new resource.
```

Comment:
"Show the form for creating a new resource." This description is generated by Laravel, pls change it